### PR TITLE
Cob/dollar mapper

### PR DESCRIPTION
**Pull request by coburncoburn**
@antonina-cherednichenko I sketched in how to map to dollars in a couple of "TODOs". Also, I removed some inheritance and constructor args that work better with only one source.

### CODE REVIEW

**Comment by coburncoburn** [comment]
also managed to commit some merge conflicts on my own comments on these todos 🤔

File: contracts/DelFiPrice.sol [hunk 1]
```diff
@@ -2,43 +2,43 @@ pragma solidity ^0.6.6;
...
+        address cBatAddress;
         address cZrxAddress;
+        address cSaiAddress;
+        address cUsdtAddress;
     }
 
-    /// @notice The reporter address whose prices checked against the median for safety
-    address anchor;
+    /// @notice The Price Oracle Proxy address whose prices checked against the source for safety
+    AnchorPriceOracle immutable anchor;
```
Comment (antonina-cherednichenko) [inline contracts/DelFiPrice.sol:36]: great!

File: contracts/DelFiPrice.sol [hunk 2]
```diff
@@ -121,68 +121,64 @@ contract DelFiPrice is OpenOracleView {
...
+            uint64 sourcePrice = priceData.getPrice(source, symbol);
+
+            // get price from anchor, and convert to dollars
+<<<<<<< HEAD
+            // TODO: get decimals right
+
+=======
+            // TODO do decimals correclty
+>>>>>>> sketch in dollar mapping
+            uint64 anchorPrice = uint64(anchor.getUnderlyingPrice(getCTokenAddress(symbol)) / usdcPrice);
```
Comment (antonina-cherednichenko) [inline contracts/DelFiPrice.sol:141]: that's nice, probably also safe with precision now

File: contracts/DelFiPrice.sol [hunk 3]
```diff
@@ -55,14 +55,14 @@ contract DelFiPrice is OpenOracleView {
     /// @notice The binary representation for 'REP' symbol, used for string comparison
     bytes32 constant symbolRep = keccak256(abi.encodePacked("REP"));
 
-    /// @notice The binary representation for 'BTC' symbol, used for string comparison
-    bytes32 constant symbolWbtc = keccak256(abi.encodePacked("BTC"));
+    /// @notice The highest ratio of the new source price to the anchor price that will still trigger the stored price to be updated
+    uint256 immutable upperBoundAnchorRatio;
 
-    /// @notice The binary representation for 'BAT' symbol, used for string comparison
-    bytes32 constant symbolBat = keccak256(abi.encodePacked("BAT"));
```
Comment (antonina-cherednichenko) [inline contracts/DelFiPrice.sol:62]: we probably should still keep these constants for mapping.

Comment (coburncoburn) [reply contracts/DelFiPrice.sol:62]: mm yes probably merged it wrong

**Pull request closed without merging**